.gitignore

```diff
@@ -1,6 +1,13 @@
 # Temporary Build Files
 build/_output
 build/_test
+# Goland
+.idea
+# Vendors OWNERS files
+vendor/**/OWNERS
+# Test Infra Dev Stats
+vendor/knative.dev/test-infra/devstats
+
```

Comment:
why is that ?

Reply:
I would say not change this .gitignore by far.
The script under /hack/update-deps.sh is supposed to update the dependencies, and remove the useless files, including OWNERS. However, some packages may keep them, based on the configuration in Gopkg.toml. Let's keep the dependencies(with all the files) based what is generated by /hack/update-deps.sh.

Reply:
Removing those.